src/main/antlr4/imports/mysql_partition.g4

```diff
@@ -0,0 +1,68 @@
+grammar mysql_partition;
+
+import mysql_idents;
+
+partition_by:
+  PARTITION BY
+  partition_by_what
+  partition_count?
+  subpartition_by?
+  partition_definitions?
```

Comment:
`skip_parens` doesn't work here? guess it could mach something that it shouldn't?